src/client.rs

```diff
@@ -195,7 +196,9 @@ impl RequestBuilder {
     /// # Ok(())
     /// # }
     /// ```
-    pub fn header<H: ::header::Header + ::header::HeaderFormat>(mut self, header: H) -> RequestBuilder {
+    pub fn header<H: ::header::Header + ::header::HeaderFormat>(mut self,
+                                                                header: H)
+                                                                -> RequestBuilder {
```

Comment:
I don't like the arguments being broken up here like this. Perhaps using a `where` clause here would allow that to be on a different line anyways...